app/lib/renderer/markdown_file_renderer.rb

```diff
@@ -37,7 +37,9 @@ def raw_content
     def preproccessed_markdown
       @preproccessed_markdown ||= begin
         removing_pagesetting_notation = raw_content.gsub(/\$\[=[=sp]=\]/, '')
-        without_metadata(removing_pagesetting_notation.each_line)
+        # Any lines that include $[//] will be stripped from the raw markdown. Use these to make comments
+        removing_comments = removing_pagesetting_notation.each_line.reject { |line| line.include?('$[//]') }
```

Comment:
I mean, I don't understand what is happening, but I don't quite understand what the removin_pagesetting_notation bit does. looks for those special symbols, but will it find `//` then?

Reply:
The pacesetting one searches for `$[===]`, `$[=s=]` and `$[=p=]` and wherever they appear, just replaces them with the empty string.

Cos it's using a regex, there are a lot of slashes involved—we have to escape `$`, `[` and `]`.

Once we've removed them, then we find all the lines that include `$[//]` and just delete the lines. Could probably merge those two things into one, but I think it's nice to keep it separate so that it's possible to follow what it says.